src/pages/auth/index.tsx

```diff
@@ -1,11 +1,22 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
+import axios from "axios";
 import Link from "next/link";
+import Router from "next/router";
 
 import AuthHeading from "@/components/AuthHeading";
 import BaseButton from "@/components/base/BaseButton";
 import InputField from "@/components/base/InputField";
+import { useStore } from "../store/store";
 
 export default function Auth() {
+  const getCSRFCookie = async () => {
+    await axios.get("http://localhost/sanctum/csrf-cookie");
```

Comment:
Hey @danielaWaldeck95, I've asked @ddoumecq to let me help with some code reviews so I can also start learning React related technologies. First small comment here: remember to make the `http://localhost` part dynamic before moving this to a different environment than local.